server/channels/store/sqlstore/store.go

```diff
@@ -49,8 +49,8 @@ const (
 	MySQLForeignKeyViolationErrorCode = 1452
 	PGDuplicateObjectErrorCode        = "42710"
 	MySQLDuplicateObjectErrorCode     = 1022
-	DBPingAttempts                    = 18
-	DBPingTimeoutSecs                 = 10
+	DBPingAttempts                    = 5
+	DBPingTimeoutSecs                 = 1
```

Comment:
I see that this is used in two slightly different cases. First is the actual context timeout to ping the db and the other one is more like an interval to wait before attempting to ping again. I think this is totally fine for the latter use but I wonder if for the former one could be a bit too low to potentially be causing unwanted failures (e.g. DNS resolution can take a while).

Reply:
Ahh sorry my bad, that was a debugging change I meant to revert. Only the pingAttempts was meant to be reduced to 5. 18 seemed like a weird number. Nice catch!